### DISCUSSION TITLE
Server is not running, but I did not disable it

### DISCUSSION THREAD

**Discussion by Jayinem** [rustdesk/rustdesk#9807] [General]
### Bug Description

This has been happening more and more lately. I go to connect from one computer to another and it says the other one is offline. The only solution is that the server is not started. The issue is I never chose to stop it. Just now both computers servers were offline and I never go in and disable them. It is happening on it's own somehow. 

### How to Reproduce

Not sure if you weren't having this issue how to reproduce it. 

### Expected Behavior

The servers continue to run 

### Operating system(s) on local side and remote side

Windows 10

### RustDesk Version(s) on local side and remote side

1.3.2 

### Screenshots

None available

### Additional Context

None

**Comment by Facefoda-se** [comment]
me too, any solution?
